### PR TITLE
BDS scan improvements.

### PR DESCRIPTION
**Pull request by seriva**
Updated gen-licences.py script to create dependencies.xml to avoid scanning all python dependencies during the BDS scan.

### CODE REVIEW

File: core/src/epicli/gen-licenses.py [hunk 1]
```diff
@@ -114,5 +114,34 @@ def _main() -> None:
...
+		<source></source>
+	</component>"""
+        count=count+1
+
+    dependencies_content = dependencies_content + """
+</components>
+"""
+    path = os.path.join(os.path.dirname(__file__), 'dependencies.xml')
+    with open(path, 'w') as file:
+        file.write(dependencies_content)
```
Comment (to-bar) [inline core/src/epicli/gen-licenses.py:144]: Using `textwrap.dedent` we could indent multiline strings, for example:
```
    dependencies_content = """\
    <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
    <components xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">"""

    count = 1
    for dep in all_deps_data:
        dep_name = dep['Name']
        dep_version = dep['Version']
        dep_website = dep['Home-page']
        dep_license = dep['License']
        dependencies_content = dependencies_content + f"""
        <component>
            <id>{count}</id>
            <name>{dep_name}</name>
            <version>{dep_version}</version>
            <license>{dep_license}</license>
            <url>{dep_website}</url>
            <source></source>
        </component>"""
        count=count+1

    dependencies_content = dependencies_content + """
    </components>
    """
    path = os.path.join(os.path.dirname(__file__), 'dependencies.xml')
    with open(path, 'w') as file:
        file.write(textwrap.dedent(dependencies_content).replace("    ", "\t"))
```

Comment (seriva) [reply core/src/epicli/gen-licenses.py]: Good suggestion:)

File: core/src/epicli/gen-dependency-info.py [hunk 2]
```diff
@@ -83,36 +84,63 @@ def _main() -> None:
...
+        dependencies_content = dependencies_content + f"""
+        <component>
+            <id>{count}</id>
+            <name>{dep_name}</name>
+            <version>{dep_version}</version>
+            <license>{dep_license}</license>
+            <url>{dep_website}</url>
+            <source></source>
+        </component>"""
+        count=count+1
```
Comment (to-bar) [inline core/src/epicli/gen-dependency-info.py:132]:
```suggestion
        count = count + 1
```

**Review by to-bar — APPROVED** [review]

**Review by atsikham — APPROVED** [review]
There is no newline, but it's not related to changes.

**Pull request merged**